Library/Formula/swiftgen.rb

```diff
@@ -0,0 +1,41 @@
+class Swiftgen < Formula
+  desc "Collection of Swift tools to generate Swift code"
+  homepage "https://github.com/AliSoftware/SwiftGen"
+  url "https://github.com/AliSoftware/SwiftGen/archive/0.4.4.tar.gz"
+  sha256 "429e50b33d1cd3e34fa81b5becff865ce5663284380e91602783bfa109c96b83"
+  head "https://github.com/AliSoftware/SwiftGen.git"
+
+  depends_on :xcode => "7.0"
+
+  def install
+    # Install all the binaries
+    ENV["SWIFTGEN_VERSION"] = "#{version} (via Homebrew)"
+    rake "install[#{bin},/]"
+
+    # Install the various test fixtures in pkgshare needed for `brew test`
+    pkgshare.install(
+      "Tests/Assets/fixtures/Images.xcassets" => "Images.xcassets",
+      "Tests/Assets/expected/FileDefaults.swift.out" => "assets.swift",
+      "Tests/Colors/fixtures/colors.txt" => "colors.txt",
+      "Tests/Colors/expected/FileDefaults.swift.out" => "colors.swift",
+      "Tests/L10n/fixtures/Localizable.strings" => "Localizable.strings",
+      "Tests/L10n/expected/FileWithDefaults.swift.out" => "l10n.swift",
+      "Tests/Storyboard/fixtures/Message.storyboard" => "Message.storyboard",
```

Comment:
Ah, that's because of how your `pkgsare.install` is implemented in Ruby — and it only works if you group the keys having a value in the end. _(in practice this passes 5 arguments to `pkgshare.install`, the first 4 ones being the un-renamed path strings, the last one being the hash with 4 keys and values; and the implementation is done in a way that can handle it)_